docs/README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Rustc Version 1.42+]][rustc] [Join us on LogDNA's Public Slack]
 
-The LogDNA agent is a blazingly fast, resource-efficient log collection client, that forwards logs to [LogDNA]. This version of the agent is written in [Rust] to ensure maximum performance, and when coupled with LogDNA's web application, provides a powerful log management tool for distributed systems, including [Kubernetes] clusters.
+The LogDNA agent is a fast, resource-efficient log collection client that forwards logs to [LogDNA]. This version of the agent is written in [Rust] to ensure maximum performance, and when coupled with LogDNA's web application, provides a powerful log management tool for distributed systems, including [Kubernetes] clusters.
```

Comment:
Should these changes be applied to the root README file as well?

Reply:
Yes... though I didn't realize there was a readme.md in the root dir. I originally assumed this one was in root, but it is in /docs. Which isn't maybe typical?